element/serialize.js

```diff
@@ -70,6 +84,51 @@ const SELF_CLOSING_TAGS = new Set( [
 	'wbr',
 ] );
 
+/**
+ * Inline element tags.
+ *
+ * https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements
+ *
+ * [ ...document.querySelectorAll( '.threecolumns code' ) ]
+ *     .map( ( el ) => el.textContent.replace( /(^<|>$)/g, '' ) )
```

Comment:
I'm not quite following this comment and how it is related.

Reply:
>I'm not quite following this comment and how it is related.

In case anyone is curious how the set is determined, or in case it ever needs to be updated in the future (assuming the structure of the referenced documentation page doesn't change).

I'll add a bit more clarity to the comment, as I did with "Extracted from" in the block validator which has similar comments:

https://github.com/WordPress/gutenberg/blob/7ad3fa44230a664228bfafba14516f70441d3d96/blocks/api/validation.js#L84-L91